packages/kit/src/node/index.js

```diff
@@ -56,6 +57,7 @@ export async function getRequest(base, req) {
 	if (req.httpVersionMajor === 2) {
 		// we need to strip out the HTTP/2 pseudo-headers because node-fetch's
 		// Request implementation doesn't like them
+		// TODO is this still true with undici?
```

Comment:
maybe it'd be clearer to say Node 18 for people who aren't familiar with Undici?
```suggestion
		// TODO is this still true with Node 18
```